language/Dutch/strings.po

```diff
@@ -876,197 +887,197 @@ msgstr "Plugins"
 #: src/frontend/mame/ui/selgame.cpp:856
 #, c-format
 msgid "Romset\t%1$-.100s\n"
-msgstr ""
+msgstr "Romset\t%1$-.100s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:857
 #, c-format
 msgid "Year\t%1$s\n"
-msgstr ""
+msgstr "Jaar\t%1$s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:858
 #, c-format
 msgid "Manufacturer\t%1$-.100s\n"
-msgstr ""
+msgstr "Fabrikant\t%1$-.100s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:862
 #, c-format
 msgid "Driver is Clone of\t%1$-.100s\n"
-msgstr ""
+msgstr "Driver is variant van\t%1$-.100s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:864
 msgid "Driver is Parent\t\n"
-msgstr ""
+msgstr "Driver is hoofdsoftware\t\n"
 
 #: src/frontend/mame/ui/selgame.cpp:867
 msgid "Analog Controls\tYes\n"
-msgstr ""
+msgstr "Analoge besturing\tJa\n"
 
 #: src/frontend/mame/ui/selgame.cpp:869
 msgid "Keyboard Inputs\tYes\n"
-msgstr ""
+msgstr "Toetensenbordinvoer\tJa\n"
 
 #: src/frontend/mame/ui/selgame.cpp:872
 msgid "Overall\tNOT WORKING\n"
-msgstr ""
+msgstr "Status\tNIET WERKEND\n"
 
 #: src/frontend/mame/ui/selgame.cpp:874
 msgid "Overall\tUnemulated Protection\n"
-msgstr ""
+msgstr "Status\tOngeëmuleerde beveiliging\n"
 
 #: src/frontend/mame/ui/selgame.cpp:876
 msgid "Overall\tWorking\n"
-msgstr ""
+msgstr "Status\tWerkend\n"
 
 #: src/frontend/mame/ui/selgame.cpp:879
 msgid "Graphics\tUnimplemented\n"
-msgstr ""
+msgstr "Beeld\tOngeïmplementeerd\n"
 
 #: src/frontend/mame/ui/selgame.cpp:881
 msgid "Graphics\tWrong Colors\n"
-msgstr ""
+msgstr "Beeld\tVerkeerde kleuren\n"
 
 #: src/frontend/mame/ui/selgame.cpp:883
 msgid "Graphics\tImperfect Colors\n"
-msgstr ""
+msgstr "Beeld\tOnvolledige kleuren\n"
 
 #: src/frontend/mame/ui/selgame.cpp:885
 msgid "Graphics\tImperfect\n"
-msgstr ""
+msgstr "Beeld\tOnvolledig\n"
 
 #: src/frontend/mame/ui/selgame.cpp:887
 msgid "Graphics\tOK\n"
-msgstr ""
+msgstr "Graphics\tOK\n"
```

Comment:
Why the inconsistent translation on this string?  You've used "Beeld" elsewhere, here you've carried across the English "Graphics".